nslsii/__init__.py

```diff
@@ -197,6 +204,17 @@ def configure_base(
             exception_logger=log_exception, ipython=get_ipython()
         )
 
+    if publish_documents_to_kafka:
+        subscribe_kafka_publisher(
+            RE,
+            beamline_name=broker_name,
+            bootstrap_servers="cmb01:9092,cmb02:9092,cmb03:9092",
+            producer_config={
+                "enable.idempotence": True,
+                "linger.ms": 0
```

Comment:
This is the default, right? https://docs.confluent.io/current/installation/configuration/producer-configs.html#linger.ms